ig/input/fsh/PopulationRecommendation.fsh

```diff
@@ -12,13 +13,45 @@ Description: ""
 * extension contains
   ageStart 0..1 MS and
   ageEnd 0..1 MS
-* extension[ageStart].value[x] only Quantity
-* extension[ageStart].value[x] obeys age-1
-* extension[ageStart].value[x].code from http://hl7.org/fhir/ValueSet/all-time-units
+* extension contains VPPopulationRecommendationLocationExtension named location 0..* MS
```

Comment:
Just to clarify:
A PopulationRecommendation that should be applied independent of any location doesn't have a location, right?

Reply:
Yes, that's correct. Likewise not providing `ageStart` or `ageEnd` implies that there is no start/end restriction on the age of the patient.

Reply:
Okay, great! Thanks!